CHANGELOG.md

```diff
@@ -2,6 +2,43 @@
 
 This changelog follows the format defined here: https://keepachangelog.com/en/1.0.0/
 
+## [1.2.0] - 2020-03-14
+
+A more detailed account of everything included is included in the dev to master PR for this release: https://github.com/ucb-bar/chipyard/pull/418
+
+### Added
+* FireMarshal now generates an extra copy of linux kernel with dwarf debugging info for use in FirePerf (#427)
+* Ring Topology System Bus NoC (#461 )
+* Add option to use blackboxed SimDRAM instead of SimAXIMem (#449)
+* Log `init-submodules` script (#433 )
+* Integration of the Ariane core into Chipyard (#448)
+* Moved the Docker image used for CI into Chipyard (prev. in BOOM) (#463)
+
+### Changed
+* IOBinders and BuildTop unification between FireChip and Chipyard (#390)
+* Update to CircleCI 2.1 config. syntax and cleanup CI file (#421)
+* Bump FireSim to version 1.9 - Includes FirePerf TracerV stack unwinding features
```

Comment:
```suggestion
* Bump FireSim to version 1.8.1 - Includes FirePerf TracerV stack unwinding features
```

Reply:
AFAIK, the decision was to keep it 1.9 @sagark @davidbiancolin

Reply:
yes, but this should be 1.9.0